packages/docusaurus-plugin-openapi/src/openapi/openapi.ts

```diff
@@ -209,8 +208,10 @@ export async function readOpenapiFiles(
   _options: {}
 ): Promise<OpenApiFiles[]> {
   if (isURL(openapiPath)) {
-    const { data } = await axios.get(openapiPath);
-    if (!data) {
+    let data;
+    try {
+      data = await (await fetch(openapiPath)).json();
```

Comment:
Can we split this into 2 lines? It’s a bit hard to read

Reply:
done